on-chain/contracts/TAParser.sol

```diff
@@ -52,7 +52,7 @@ contract TAParser {
     {
         TA memory ta = decodeTA(taData);
 
-        FnCallClaims memory claims = decodeAPICallClaims(ta.Data);
+        FnCallClaims memory claims = decodeFnCallClaims(ta.Data);
```

Comment:
I udpated the function names since these are for `attest-fn-call` not `attest-api-call`